charts/stash-enterprise/templates/cluster-role.yaml

```diff
@@ -67,7 +67,7 @@ rules:
   - secrets
   - services
   - endpoints
-  verbs: ["get"]
+  verbs: ["get","list","create","patch"]
```

Comment:
Why do we need this?

Reply:
We create the docker secret from the operator in the workload namespace.

Reply:
Let's separate our the secrets part and just give the extra permission for secrets, not for services and endpoints.

Reply:
done